app/webpacker/stylesheets/components/pages/certification/_cs-accelerator.scss

```
@@ -11,8 +11,18 @@
   background-color: $concrete;
 }
 
-.ncce-programmes-activity__title {
-  color: unset;
+.ncce-progress-section {
+  display: flex;
+  flex: wrap;
+  flex-direction: column;
+
+  @include govuk-media-query($from: 770px) {
+    flex-direction: row;
+  }
```

```suggestion
  @include govuk-media-query($from: desktop) {
    flex-direction: row;
  }
```
is usually much better to use the named variables over hard-coded magic numbers. Can you fix this pls?

---

resolved too thank you